### PR TITLE
Creating one more example which only uses redux and no thunk as middl…

### PR DESCRIPTION
**Pull request by ravindermahajan890**
Adding one more example which only uses redux and not thunk. This helps a lot for a beginner to understand basic redux first.

### CODE REVIEW

File: examples/with-redux-thunk/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/zeit/next.js/tree/master/examples/with-redux-thunk)
+
+# Redux example
```
Comment (ijjk) [inline examples/with-redux-thunk/README.md]:
```suggestion
# Redux Thunk example
```

File: examples/with-redux-thunk/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Deploy it to the cloud with [now](https://zeit.co/now) ([download](https://zeit.co/download))
+
+```bash
+now
+```
+
+## The idea behind the example
+
+This example shows how to integrate Redux in Next.js.
```
Comment (ijjk) [inline examples/with-redux-thunk/README.md]:
```suggestion
This example shows how to integrate Redux and Redux Thunk in Next.js.
```

Comment (ravindermahajan890) [reply examples/with-redux-thunk/README.md]: Fixed.

File: examples/with-redux-thunk/README.md (hunk 1 above)
Comment (ravindermahajan890) [reply examples/with-redux-thunk/README.md]: Fixed

**Review by ijjk — APPROVED** [review]

**Pull request merged**